src/test/java/api/requests/RequestsAPILocationTests.java

```diff
@@ -52,27 +51,21 @@ public void locationIsIncludedForSingleRequest()
       .forItem(smallAngryPlanet)
       .by(requester));
 
-    JsonObject representation = request.getJson();
-
-    assertThat("has item location",
-      representation.getJsonObject("item").containsKey("location"), is(true));
+    JsonObject createdRequest = request.getJson();
 
-    assertThat(
-      representation.getJsonObject("item").getJsonObject("location").getString("name"),
```

Comment:
Why has this check for the location name been removed?

Reply:
It's now inside of hasItemLocationProperties()

Reply:
As far as I can tell, the `hasItemLocationProperties` matcher only `library name` and the `location code` and not the `location name`

Reply:
The name check has been moved to hasItemLocationProperties ()